### ISSUE TITLE
Audio is not working

### ISSUE THREAD

**Issue by bernard01** [OpenShot/openshot-qt#4464]
Looks like there is already an issue https://github.com/OpenShot/openshot-qt/issues/1975 but this is closed. So I create a new issue.

I have downloaded the software for the first time, fresh 2.6.1 version so all settings are default. 

OS is Windows 7.

I can play MP3 and MP4 video files with VLC media player where the default output goes to my earplugs or speakers consistent with system settings. I cannot make a video without audio. Which is critical. This is really wonderful software. Something must be missing. How do other people do it?

**Comment by dodgePT** [comment]
I'm also on WIndows (10) and the previous version was working great. Updating to 2.6.1 broke everything.
I was editing a project from 2.6.0 and now, video plays in slow motion and there's no audio.
I tried all kinds of cache options, to no avail. Uninstalled and reinstalled. I have a top tier PC and while previewing, the CPU goes to 6% at most, but the video keeps playing in slomo, something is very wrong with the latest release.

**Comment by bernard01** [comment]
I have followed instructions of that 100% relevant video:
https://www.reddit.com/r/OpenShot/comments/gmi93p/how_to_fix_stuttering_video_preview_openshot/
and could not get any improvements even with 15 frames per second as suggested. I have also followed instructions in another video that suggested to increase cache RAM. I have a 8 core intel I7 desktop with dedicated video card, and CPU usage is under 5%.
Easiest way to verify is to play the video from file where no time line is involved, shows you that OpenShot does not have a functioning video player at the most basic level. Solution: Uninstall software, use something else. This guy does not know how to write software. It is a gimmick.
